protocol/triple/internal/proto/greettriple/greet.triple.go

```diff
@@ -0,0 +1,385 @@
+// problem by either regenerating this code with an older version of connect or updating the connect
+// version compiled into your binary.
+const _ = triple_protocol.IsAtLeastVersion0_1_0
+
+const (
+	// GreetServiceName is the fully-qualified name of the GreetService service.
+	GreetServiceName = "greet.GreetService"
+)
+
+// These constants are the fully-qualified names of the RPCs defined in this package. They're
+// exposed at runtime as Spec.Procedure and as the final two segments of the HTTP route.
+//
+// Note that these are different from the fully-qualified method names used by
+// google.golang.org/protobuf/reflect/protoreflect. To convert from these constants to
+// reflection-formatted method names, remove the leading slash and convert the remaining slash to a
+// period.
+const (
+	// GreetServiceGreetProcedure is the fully-qualified name of the GreetService's Greet RPC.
+	GreetServiceGreetProcedure = "/greet.GreetService/Greet"
+	// GreetServiceGreetStreamProcedure is the fully-qualified name of the GreetService's GreetStream
+	// RPC.
+	GreetServiceGreetStreamProcedure = "/greet.GreetService/GreetStream"
+	// GreetServiceGreetClientStreamProcedure is the fully-qualified name of the GreetService's
+	// GreetClientStream RPC.
+	GreetServiceGreetClientStreamProcedure = "/greet.GreetService/GreetClientStream"
+	// GreetServiceGreetServerStreamProcedure is the fully-qualified name of the GreetService's
+	// GreetServerStream RPC.
+	GreetServiceGreetServerStreamProcedure = "/greet.GreetService/GreetServerStream"
+)
+
+// GreetServiceClient is a client for the greet.GreetService service.
+type GreetServiceClient interface {
+	Greet(ctx context.Context, req *proto.GreetRequest, opts ...client.CallOption) (*proto.GreetResponse, error)
+	GreetStream(ctx context.Context, opts ...client.CallOption) (GreetService_GreetStreamClient, error)
+	GreetClientStream(ctx context.Context, opts ...client.CallOption) (GreetService_GreetClientStreamClient, error)
+	GreetServerStream(ctx context.Context, req *proto.GreetServerStreamRequest, opts ...client.CallOption) (GreetService_GreetServerStreamClient, error)
+}
+
+// NewGreetServiceClient constructs a client for the greet.GreetService service. By default, it uses
+// the Connect protocol with the binary Protobuf Codec, asks for gzipped responses, and sends
+// uncompressed requests. To use the gRPC or gRPC-Web protocols, supply the connect.WithGRPC() or
+// connect.WithGRPCWeb() options.
+//
+// The URL supplied here should be the base URL for the Connect or gRPC server (for example,
+// http://api.acme.com or https://acme.com/grpc).
+func NewGreetServiceClient(cli *client.Client) (GreetServiceClient, error) {
+	if err := cli.Init(&GreetService_ClientInfo); err != nil {
+		return nil, err
+	}
+	return &GreetServiceClientImpl{
+		cli: cli,
+	}, nil
+}
+
+func SetConsumerService(srv common.RPCService) {
+	config.SetClientInfoService(&GreetService_ClientInfo, srv)
+}
```

Comment:
How about we remove all the unnecessary old code stubs from the generated file?
For users who rely on old stubs, they can still use the old compiler to generate the code.

Reply:
The code generated by old compiler still depends on grpc library. Maybe there is a scenario that user wants to upgrade all services with new compiler? In this case, user also needs a classical config loading way to solve the complex configuration problem.(Assume that we have not provided NewClient with dubbogo.yml)